src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+
```

Comment:
no need to remove this line
```suggestion
 
 const newSeparator = String(separator);
```

Reply:
and use newSeparator in loop